examples/federated-connections-with-access-tokens/chatbot/package.json

```diff
@@ -11,7 +11,9 @@
   "workspaces": [
     "./server",
     "./client",
-    "./shared"
+    "./shared",
+    "../../../packages/ai",
+    "../../../packages/ai-vercel"
```

Comment:
temporary change to ensure these local packages are bun installed correctly. If we were to keep this as a nested bun workspace, we could get away with npm installing the deployed versions instead. Another option may be to remove bun and prefer npm to pick up the workspace references automatically (using package glob/ `*` pattern like the others here)